src/components/PostPageDetail/index.js

```diff
@@ -678,7 +675,7 @@ const PostPageDetailIdComponent = (props) => {
   );
 };
 
-export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
+export default PostPageDetailIdComponent;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withInteractionsManaged` HOC has been removed from the export statement. As mentioned earlier, ensure that this doesn't affect the component's behavior.
```diff
- export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
+ export default PostPageDetailIdComponent;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->